Gulpfile.js

```diff
@@ -16,6 +16,7 @@ var APP_DIR             = path.join(__dirname, 'test/data/test-app-regular');
 var ASAR_ARCHIVE_PATH   = path.join(__dirname, 'test/data/test-app.asar');
 var CONFIG_PATH_REGULAR = path.join(__dirname, 'test/data/app-config-regular');
 var CONFIG_PATH_ASAR    = path.join(__dirname, 'test/data/app-config-asar');
+var CONFIG_PATH_JS      = path.join(__dirname, 'test/data/app-config-js');
```

Comment:
`require` won't be able to load a module without the correct extension (`.js`). I've added a more detailed comment to our existing conversation: https://github.com/DevExpress/testcafe-browser-provider-electron/pull/54#discussion_r361151757